uportal-war/src/main/resources/properties/contexts/applicationContext.xml

```diff
@@ -33,15 +33,19 @@
     <aop:aspectj-autoproxy/>
     
     <context:annotation-config/>
+    <context:component-scan base-package="org.jasig.portal.portlets">
+        <context:include-filter type="regex" expression=".*\.PortletPreferencesJsonDao" />
+        <context:include-filter type="regex" expression=".*\.account\.UserAccountHelper" />
+    </context:component-scan>
     <context:component-scan base-package="org.jasig.portal">
-        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.portlets\."/>
-        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.security\.mvc\."/>
-        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.json\.rendering\."/>
-        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.layout\.dlm\.remoting\."/>
-        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.security\.remoting\."/>
-        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.rest\."/>
-        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.redirect\."/>
-        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.rendering\."/>
+        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.portlets\..+"/>
+        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.security\.mvc\..+"/>
+        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.json\.rendering\..+"/>
+        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.layout\.dlm\.remoting\..+"/>
+        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.security\.remoting\..+"/>
+        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.rest\..+"/>
+        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.redirect\..+"/>
+        <context:exclude-filter type="regex" expression="org\.jasig\.portal\.rendering\..+"/>
```

Comment:
Nitpick:  Generally, I think it's better to use a filter based on annotation type than by regex on the package name.   That way, you easily add Services anywhere and it will get picked up in the parent context.   Controllers can be placed anywhere and will get picked up in the portlet contex (not that you should!)

Having said that, I don't think there is anything here bad enough to hold up merge if the community is fine with this approach.